nodeEditor/src/diagram/graphCanvas.tsx

```diff
@@ -841,14 +841,13 @@ export class GraphCanvasComponent extends React.Component<IGraphCanvasComponentP
         this.props.globalState.onRebuildRequiredObservable.notifyObservers();
     }
 
-    processEditorData(editorData: IEditorData) {
+    processEditorData(editorData: IEditorData, isImportingAFrame = false) {

```

Comment:
2nd parameter does not seem to be used anymore

Reply:
will remove!